src/App.js

```diff
@@ -1,12 +1,32 @@
-import Text from "./component/Text";
-import Button from "./component/Button";
-import TextCard from "./component/TextCard";
+import ColorCard from "./component/ColorCard";
+import TextBox from "./component/Text";
+import MediaCard from "./component/MediaCard";
+
 function App() {
   return (
     <>
-      <TextCard text="이게 TextCard...?" button="click" />
+    <MediaCard img ="https://p4.wallpaperbetter.com/wallpaper/13/870/486/puppy-backgrounds-for-desktop-hd-backgrounds-wallpaper-preview.jpg" 
+    fontSize="25px" margin_right="3.5cm" margin_right2="2.1cm"
+    tcolor2="gray" color="white" backgroundColor="#7794DF" head="귀여운 멍멍이"content="귀엽다,,우리집에도 천사있음,," width="280px"/>
```

Comment:
자바스크립트 진영에서 함수와 변수에 대한 네이밍 컨벤션은 camelCase 를 사용합니다! 아래와 같이 컨벤션을 변경해볼까요?

```suggestion
    <MediaCard img ="https://p4.wallpaperbetter.com/wallpaper/13/870/486/puppy-backgrounds-for-desktop-hd-backgrounds-wallpaper-preview.jpg" 
    fontSize="25px" marginRight="3.5cm" marginRight2="2.1cm"
    tcolor2="gray" color="white" backgroundColor="#7794DF" head="귀여운 멍멍이"content="귀엽다,,우리집에도 천사있음,," width="280px"/>
```